Pagination: add Getting Started tutorial (#6890)
DevExpress/devextreme-documentation: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md

> Integrate a [load panel](/Documentation/Guide/UI_Components/LoadPanel/Overview/) to inform users that cards are loading. This step is optional.

Reviewer: Other steps are not in imperative mood. They say "the code does this/that". 

```suggestion
The following code integrate a [load panel](/Documentation/Guide/UI_Components/LoadPanel/Overview/) into the application. The panel appears when the app requests card data from the remote service. This step is optional.
```

---
##### jQuery

> To integrate DevExtreme LoadPanel:

Reviewer: ```suggestion
To integrate the DevExtreme LoadPanel component:
```

> 1. Add LoadPanel to the code.

Reviewer: ```suggestion
1. Add a LoadPanel to the code.
```

2. [Display](/Documentation/ApiReference/UI_Components/dxLoadPanel/Methods/#show) it before calling the render function.

> 3. [Hide](/Documentation/ApiReference/UI_Components/dxLoadPanel/Methods/#hide) it after rendering each color.

Reviewer: ```suggestion
3. [Hide](/Documentation/ApiReference/UI_Components/dxLoadPanel/Methods/#hide) it after render.
```

    </html>

    <!-- tab: index.js -->
    $(() => {
        const loadPanel = $('#load-panel')
            .dxLoadPanel({
                position: {
                    my: 'top',
                    at: 'top',
                    of: '#cards',
                },
                visible: false,
                showIndicator: true,
                showPane: true,
                hideOnOutsideClick: false,
            })
            .dxLoadPanel('instance');

        const pagination = $('#pagination')
            .dxPagination({
                // ...
                onOptionChanged: (e) => {
                    if (e.name === 'pageSize' || e.name === 'pageIndex') {
                        const pageIndex = pagination.option('pageIndex');
                        const pageSize = pagination.option('pageSize');
                        loadPanel.show();
                        renderCards(pageSize, pageIndex).finally(() => loadPanel.hide());
                    }
                },
            })
        .dxPagination('instance');
        // ...
        loadPanel.show();
        renderCards(pageSize, pageIndex).finally(() => loadPanel.hide());
    });

##### Angular

> To integrate DevExtreme LoadPanel:

Reviewer: ```suggestion
To integrate the DevExtreme LoadPanel component:
```

> 1. Add LoadPanel to the code.

Reviewer: ```suggestion
1. Add a LoadPanel to the code.
```

2. [Display](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#visible) it before calling the render function.

> 3. Hide it after rendering each color.

Reviewer: ```suggestion
3. Hide it after render.
```

    <dx-pagination ... ></dx-pagination>
    <!-- ... -->
    <dx-load-panel
        [(visible)]="loadPanelVisible"
        [showIndicator]="true"
        [showPane]="true"
        [hideOnOutsideClick]="false"
    >
        <dxo-position my="top" at="top" of="#cards"></dxo-position>
    </dx-load-panel>

    <!-- tab: app.component.ts -->
    // ...
    export class AppComponent {
        // ...
        loadPanelVisible = false;

        async fetchColorsForPage(): Promise<void> {
            const startIndex = (this.pageIndex - 1) * this.pageSize;
            const endIndex = this.pageIndex * this.pageSize;
            const hexSubset = this.hexCodes.slice(startIndex, endIndex);

            const promises: Promise<Color>[] = hexSubset.map((hex) => {
                // ...
            });

            this.loadPanelVisible = true;
            try {
                const fetchedColors = await Promise.all(promises);
                this.visibleCards = fetchedColors;
            } catch (error) {
                console.error('Error fetching colors:', error);
            } finally {
                this.loadPanelVisible = false;
            }
        }
    }

##### Vue

> To integrate DevExtreme LoadPanel:

Reviewer: ```suggestion
To integrate the DevExtreme LoadPanel component:
```

> 1. Add LoadPanel to the code.

Reviewer: ```suggestion
1. Add a LoadPanel to the code.
```

2. [Display](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#visible) it before calling the render function.

> 3. Hide it after rendering each color.

Reviewer: ```suggestion
3. Hide it after render.
```

        <!-- ... -->
        <DxLoadPanel
            v-model:visible="loadPanelVisible"
            :show-indicator="true"
            :show-pane="true"
            :hide-on-outside-click="false"
        >
            <DxPosition my="top" at="top" of="#cards" />
        </DxLoadPanel>
    </template>

    <script setup lang="ts">
    // ...
    import { DxLoadPanel, DxPosition } from 'devextreme-vue/load-panel';

    const loadPanelVisible = ref(false);

    const fetchColorsForPage = async () => {
        loadPanelVisible.value = true;
        const startIndex = (pageIndex.value - 1) * pageSize.value;
        const endIndex = startIndex + pageSize.value;
        const hexSubset = hexCodes.value.slice(startIndex, endIndex);

        const promises = hexSubset.map((hex) => {
            // ...
        });

        try {
            const results = await Promise.all(promises);
            visibleCards.value = results.filter((color): color is Color => color !== null);
        } catch (error) {
            console.error('Error fetching colors:', error);
        } finally {
            loadPanelVisible.value = false;
        }
    };

    // ...
    </script>

##### React

> To integrate DevExtreme LoadPanel:

Reviewer: ```suggestion
To integrate the DevExtreme LoadPanel component:
```

> 1. Add LoadPanel to the code.

Reviewer: ```suggestion
1. Add a LoadPanel to the code.
```

2. [Display](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#visible) it before calling the render function.

> 3. Hide it after rendering each color.

Reviewer: ```suggestion
3. Hide it after render.
```
